api/jobs.go

```diff
@@ -199,6 +200,8 @@ func (api *API) GetJobsHandler(w http.ResponseWriter, req *http.Request) {
 		Offset: offset,
 		Limit:  limit,
 	}
+
```

Comment:
(suggestion): extra space

Reply:
In this particular case, I think the extra space is nice to clearly separate the setting of the `options` and then calling the function which uses this `options` afterwards